projects/laji/test/+observation/observation.po.ts

```diff
@@ -0,0 +1,208 @@
+    &countryId=ML.206&finnishMunicipalityId=ML.532&provinceId=ML.1245&elyCentreId=ML.1259&biogeographicalProvinceId=ML.251
+    &area=asd&time=2022-01-01%2F&keyword=asd&collectionId=gbif-dataset:994fb34c-ba13-4768-a238-b1d3eb35c90d&typeOfOccurrenceId=MX.doesNotOccur
+    &coordinates=63.250569:63.417915:25.523624:25.64792:WGS84:1&sourceId=KE.167,KE.3&recordBasis=PRESERVED_SPECIMEN&lifeStage=ADULT&sex=MALE
+    &documentId=asd&recordQuality=EXPERT_VERIFIED&wild=WILD&teamMemberId=1795001&primaryHabitat=MKV.habitatM%5BMKV.habitatSpecificTypeV%5D
+    &plantStatusCode=MY.plantStatusCodeL&sourceOfCoordinates=REPORTED_VALUE&finnish=true&invasive=false&typeSpecimen=true&hasDocumentMedia=true
+    &hasGatheringMedia=true&hasUnitMedia=false&secured=true&unidentified=true&needsCheck=true&breedingSite=true&useIdentificationAnnotations=false
+    &includeSubTaxa=false&annotated=true&individualCountMin=0&individualCountMax=1000&occurrenceCountFinlandMax=1000&coordinateAccuracyMax=1000
+    &qualityIssues=BOTH&firstLoadedSameOrAfter=2022-01-01&loadedSameOrAfter=2022-01-01&season=0101%2F1218
+  `;
+  public map = new MapPageObject(this.page, this.page.getByTestId('observation-map'));
+  public tabs: Record<string, LUTabPO> = {
+    list: new LUTabPO(this.page, 'list'),
+    map: new LUTabPO(this.page, 'map')
+  };
+
+  public $occurrenceCountFinlandMax: Locator = this.page.locator('input[name=occurrenceCountFinlandMax]');
+
+  public timePanel = new LUPanel(this.page, '.laji-panel-time');
+  public dateBegin = new DatePicker(this.page, '.observation-time-container laji-datepicker.time-start');
+  public dateEnd = new DatePicker(this.page, '.observation-time-container laji-datepicker.time-end');
+  public $today = this.page.locator('.btn-today');
+  public $week = this.page.locator('.btn-week');
+  public $year = this.page.locator('.btn-year');
+
+  public placePanel = new LUPanel(this.page, '.laji-panel-places');
+  public $placePanel = this.page.locator('.laji-panel-places');
+  public $drawRectangleBtn = this.page.locator('.draw-rectangle');
+  public $enterYkjBtn = this.page.locator('.enter-ykj-grid');
+  public $drawPolygonBtn = this.page.locator('.draw-polygon');
+  public $coordinateIntersectMinBtn = this.page.locator('.coordinate-intersect-min');
+  public $coordinateIntersectMaxBtn = this.page.locator('.coordinate-intersect-max');
+  public $mapSpinner = this.page.locator('.loading-map');
+  public $searchBtn = this.page.locator('.observation-search-btn');
+
+  private $activeFiltersBtn = this.page.locator('.active-filters-btn');
+  private toast = new ToastPO(this.page);
+
+  constructor(private page: Page) { }
+
+  async navigateTo(sub: 'list' | '' = '', query?: Record<string, string>) {
+    await this.page.goto(`observation/${sub}?${new URLSearchParams(query || {}).toString()}`);
+  }
+
+  async navigateToViewWithAllFilters() {
+    await this.page.goto(this.pathWithAllFilters);
+  }
+
+  async hasGraphqlErrors() {
```

Comment:
Now it should work but we had discussion on the other PR (https://github.com/luomus/laji/pull/481) that had similar thing that the `hasGraphQLApiErrors` variable can be a little confusing, maybe this could also be converted to a function like here for consistency https://github.com/luomus/laji/pull/481/commits/5351376f87eda7fe9cb8d1df24ddd6dd08d67b33